File: Selenium/Project/CRM_Activity5.java
package Project;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.support.Color;
import org.openqa.selenium.support.ui.WebDriverWait;
import org.testng.Assert;
import org.testng.annotations.AfterClass;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.Test;

public class CRM_Activity5 {
	WebDriver driver;
	WebDriverWait wait;
    
    @BeforeClass
    public void setUp() {
        //Open a browser
        driver = new ChromeDriver();
     driver.get("http://alchemy.hguy.co/crm");
    }
    
    @Test(priority = 1)
	  public void loginFormTest() {
		  //Provide username and password
		  driver.findElement(By.id("user_name")).sendKeys("admin");
		  driver.findElement(By.id("username_password")).sendKeys("pa$$w0rd");
		  //Find the submit button and click it
		  driver.findElement(By.xpath("//input[@type='submit']")).click();
		  
  }
    
    // Test function
    @Test(priority = 2)
    public void colorNavMenu() throws InterruptedException {
    	Thread.sleep(5000);
    	//Get the color of navigation menu
    	driver.manage().window().maximize();
    	Color navMenuColor = Color.fromString(driver.findElement(By.xpath("//div[@id='toolbar']")).getCssValue("color"));
    	 System.out.println("Color as RGB: " + navMenuColor.asRgb());
         System.out.println("Color as hexcode: " + navMenuColor.asHex());
    	
    }
    
    
	 @AfterClass
	    public void tearDown() {
	        driver.quit();
	    }
	    
}

File: Selenium/Project/CRM_Activity5.py
# Getting colors
# Goal: Get the color of the navigation menu

# Import webdriver from selenium
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.color import Color

# Start the Driver
with webdriver.Chrome() as driver:

    driver.get("https://alchemy.hguy.co/crm/")

# Find and select the username and password fields
# Enter login credentials into the respective fields
    driver.find_element(By.ID,"user_name").send_keys("admin")
    driver.find_element(By.ID,"username_password").send_keys("pa$$w0rd")
    driver.find_element(By.ID,"bigbutton").click()
    homePageHeader = driver.find_element(By.ID,"tab0").text
    # print(homePageHeader)
    if(homePageHeader=="SUITECRM DASHBOARD"):
        print("Home Page Opened Successfully !")
    else:
        print("Login failed !")

# Get the color of the navigation menu and print it to the console.
    nav_menu_color = Color.from_string(driver.find_element(By.ID, "toolbar").value_of_css_property("color"))
    print("Navigation Menu colour as Hexcode: ", nav_menu_color.hex)
    print("Navigation Menu colour as RGB: ", nav_menu_color.rgb)

# Close the browser
    driver.quit()